### ISSUE TITLE
Compilation failure due to mismatched function signature in ec2cluster

### ISSUE THREAD

**Issue by cdelorme** [crewjam/etcd-aws#13]
The call to [WatchLifecycleEvents](https://github.com/crewjam/etcd-aws/blob/master/lifecycle.go#L57) in the etcd-aws source is short a parameter (the queueURL) according to [ec2cluster source](https://github.com/crewjam/ec2cluster/blob/master/lifecycle.go#L81).

As a result the project does not compile.

**Comment by cdelorme** [comment]
I created a pull request: #15 

_I am not sure if we want to panic on error, or what will happen if we pass in an empty string._

**Comment by pieterlange** [comment]
@crewjam bump. Any change to get https://github.com/crewjam/etcd-aws/pull/14 or https://github.com/crewjam/etcd-aws/pull/15 merged?

**Comment by crewjam** [comment]
Yeah sorry for the delay. Travel today, I'll try to take a look today or
tomorrow.

**Comment by pieterlange** [comment]
A few days doesn't matter at this point. ;-) Thanks for the continued support on this project.

**Comment by crewjam** [comment]
Merged #14. Is it working for you now?
